.github/workflows/docker.yml

```diff
@@ -58,125 +52,19 @@ jobs:
-      - name: Login to DockerHub
-        uses: docker/login-action@v2
-        with:
-          username: ${{ secrets.DOCKERHUB_USERNAME }}
-          password: ${{ secrets.DOCKERHUB_TOKEN  }}
-      - name: Build and push
-        id: docker_build
-        uses: docker/build-push-action@v4
-        with:
-          push: true
-          pull: true
-          context: .
-          tags: ${{ steps.meta.outputs.tags }}
-          file: docker/${{ matrix.os }}/Dockerfile
-      - name: Image digest
-        run: echo ${{ steps.docker_build.outputs.digest }}
-
-
-  # Only run for push to main branch
-  # Take care of tag latest
-  # This job needs to build the configured image (ubuntu)
-  # again for main branch to create latest tag.
-  docker-main-latest:
-    name: build and push latest for main branch
-    if: github.ref == 'refs/heads/main' && github.repository_owner == 'OSGeo'
-    runs-on: ubuntu-latest
-
-    steps:
-      - name: Checkout
-        uses: actions/checkout@v3
-        with:
-          fetch-depth: 0
-      - id: meta
-        name: Create tag name
-        run: |
-          tag=${DOCKERHUB_REPOSITORY}:latest
-          echo "tags=$tag" >> $GITHUB_OUTPUT
-      - name: Log
-        run: echo ${{ steps.meta.outputs.tags }}
-      - name: Set up QEMU
-        uses: docker/setup-qemu-action@v2
-      - name: Set up Docker Buildx
-        uses: docker/setup-buildx-action@v2
-      - name: Login to DockerHub
-        uses: docker/login-action@v2
-        with:
-          username: ${{ secrets.DOCKERHUB_USERNAME }}
-          password: ${{ secrets.DOCKERHUB_TOKEN  }}
-      - name: Build and push
-        id: docker_build
-        uses: docker/build-push-action@v4
-        with:
-          push: true
-          pull: true
-          context: .
-          tags: ${{ steps.meta.outputs.tags }}
-          file: docker/ubuntu/Dockerfile
-      - name: Image digest
-        run: echo ${{ steps.docker_build.outputs.digest }}
-
-
-  # run for releases, take care of release tags
-  docker-release-os-matrix:
-    name: build and push release for ${{ matrix.os }}
-    if: startsWith(github.ref, 'refs/tags/') && github.repository_owner == 'OSGeo'
-    runs-on: ubuntu-latest
-    strategy:
-      matrix:
-        os:
-          - alpine
-          - debian
-          - ubuntu
-      fail-fast: false
-
-    steps:
-      - name: Checkout
-        uses: actions/checkout@v3
-        with:
-          fetch-depth: 0
-      - name: Create image and tag name
+      - name: Docker meta
         id: meta
         uses: docker/metadata-action@v4
         with:
-          # images: ${DOCKERHUB_REPOSITORY}
           images: osgeo/grass-gis
           tags: |
             type=ref,event=tag
+            type=ref,event=branch
+            type=raw,value=current,enable=${{ github.ref == format('refs/heads/{0}', 'releasebranch_8_3') }}
+            type=raw,value=latest,enable=${{ startsWith(github.ref, 'refs/tags/') && matrix.os == 'ubuntu' }},suffix=
```

Comment:
I would proceed with this as in the past - this one file for all active release branches. So now it contains 8.3 and will be backported. Once 8.4 comes out, this needs to be changed (as before) and backported as well. So for older branches the conditions would then not fit anymore.
If a better solution comes up, it can be approached in a different PR - as this PR is not introducing this poblem but only not solving it.

Reply:
Right, if we backport to all active branches, then the code is fine, that is with the `refs/tags/8.3` suggestion.